### PR TITLE
Duplicate id removal

### PR DESCRIPTION
**Pull request by BharadwajSairam**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the enable_keep_helm label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/common/respondent-summary/form.njk [hunk 1]
```diff
@@ -47,7 +47,7 @@
             <div class="govuk-form-group">
           <div class="govuk-checkboxes govuk-!-padding-bottom-2">
             <div class="govuk-checkboxes__item">
-              <input class="govuk-checkboxes__input" style="background-color: white" id="declarationCheck" name="declarationCheck" type="checkbox" value="declaration">
+              <input class="govuk-checkboxes__input" style="background-color: white" id="declarationCheckInput" name="declarationCheck" type="checkbox" value="declaration">
```
Comment (vivek-sekhar) [inline src/main/steps/common/respondent-summary/form.njk:50]: Why is the id changed? Is the id used in code any where? Also you have not associated the input and label

File: src/main/steps/common/form/form.njk [hunk 2]
```diff
@@ -107,7 +107,7 @@
             name: "saveAndComeLater",
             value: true,
             attributes: {
-              id: 'main-form-save-as-draft'
+              id: 'main-form-save-and-come-back-later'
```
Comment (vivek-sekhar) [inline src/main/steps/common/form/form.njk:110]: Why is the id changed? Is the id used in code any where? Does it impact any of the functionality? Have you tested the changes?

File: src/main/steps/common/form/form.njk [hunk 3]
```diff
@@ -93,7 +93,7 @@
             name: "onlycontinue",
             value: true,
             attributes: {
-              id: 'main-form-save-as-draft'
+              id: 'main-form-only-continue'
```
Comment (vivek-sekhar) [inline src/main/steps/common/form/form.njk:96]: Why is the id changed? Is the id used in code any where? Does it impact any of the functionality? Have you tested the changes?

File: src/main/steps/common/form/form.njk [hunk 4]
```diff
@@ -52,7 +52,7 @@
             disabled: true if form.submit.disabled else false,
             preventDoubleClick: true,
             attributes: {
-              id: 'main-form-submit'
+              id: 'main-form-save-and-continue'
```
Comment (vivek-sekhar) [inline src/main/steps/common/form/form.njk:55]: Why is the id changed? Is the id used in code any where? Does it impact any of the functionality? Have you tested the changes?

File: src/main/steps/common/form/form.njk [hunk 5]
```diff
@@ -23,7 +23,7 @@
             value: true,
             preventDoubleClick: true,
             attributes: {
-              id: 'main-form-submit'
+              id: 'main-form-access-code-check'
```
Comment (vivek-sekhar) [inline src/main/steps/common/form/form.njk:26]: Why is the id changed? Is the id used in code any where? Does it impact any of the functionality? Have you tested the changes?

**Comment by BharadwajSairam** [comment]
Created https://github.com/hmcts/prl-citizen-frontend/pull/1653 new PR so closing this

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/common/form/form.njk (hunk 2 above)
Comment (BharadwajSairam) [reply src/main/steps/common/form/form.njk:110]: Its duplicated in same file so changed it 
Its not used anywhere across the application

File: src/main/steps/common/form/form.njk (hunk 3 above)
Comment (BharadwajSairam) [reply src/main/steps/common/form/form.njk:96]: Its duplicated in same file so changed it
Its not used anywhere across the application

File: src/main/steps/common/form/form.njk (hunk 5 above)
Comment (BharadwajSairam) [reply src/main/steps/common/form/form.njk:26]: Its duplicated in same file so changed it
Its not used anywhere across the application

File: src/main/steps/common/form/form.njk (hunk 4 above)
Comment (BharadwajSairam) [reply src/main/steps/common/form/form.njk:55]: Its duplicated in same file so changed it
Its not used anywhere across the application

File: src/main/steps/common/respondent-summary/form.njk (hunk 1 above)
Comment (BharadwajSairam) [reply src/main/steps/common/respondent-summary/form.njk:50]: name and for are associated right?
should it be id and for ?

**Pull request closed without merging**